include/quic/btstream.hpp

```diff
@@ -36,7 +36,7 @@ namespace oxen::quic
         std::weak_ptr<BTRequestStream> return_sender;
         ConnectionID cid;
 
-        message(BTRequestStream& bp, bstring req, bool is_error = false);
+        message(BTRequestStream& bp, bstring req, bool is_timeout = false);
 
       public:
         void respond(bstring_view body, bool error = false);
```

Comment:
Discussion point:
Currently our response types are "errors" ("E") and "responses" ("R"), but we could introduce a "timeout" ("T") option to actually make a useful differentiation between the two in the application code. Otherwise, we'd have to use some standardized "timeout response" that could be parsed by the library code

Reply:
Timeout is not a response; it happens because we didn't get *any* response to the request within a certain amount of time.